search_service/api/filter.py

```diff
@@ -0,0 +1,95 @@
+from http import HTTPStatus
+from typing import Any, Dict, Iterable  # noqa: F401
+
+from flask_restful import Resource, reqparse
+from flasgger import swag_from
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+from search_service.api.table import TABLE_INDEX
+from search_service.models.table import SearchTableResultSchema
+from search_service.models.dashboard import SearchDashboardResultSchema
+from search_service.proxy import get_proxy_client
+
+
+class BaseFilterAPI(Resource):
```

Comment:
Suggestion: how about separate this class into separate file? It just because this file would grow as we introduce more filterable resource.